crates/turborepo-telemetry/src/lib.rs

```diff
@@ -402,14 +415,16 @@ mod tests {
 
     #[tokio::test]
     async fn test_closing() {
+        let (_, temp_dir) = temp_dir();
```

Comment:
Careful with this, `_` will cause the var to be dropped immediately vs `_tmp` which will drop when it goes out of scope.
```suggestion
        let (_, temp_dir) = temp_dir();
```

Reply:
Oh huh, I didn't realize `_` was special. Sure enough: https://doc.rust-lang.org/beta/nightly-rustc/rustc_lint/let_underscore/static.LET_UNDERSCORE_DROP.html#:~:text=Statements%20which%20assign%20an%20expression,duration%20of%20an%20entire%20scope.

Good catch.

Reply:
Updated.